### ISSUE TITLE
PKGI loader freezes when downloading, will not show further progress

### ISSUE THREAD

**Issue by diddyrips5000** [mmozeiko/pkgi#45]
When downloading a game over 1.5GB, the loader freezes (small progress bar/info screen), this happens on all the games over 1.5GB+ and thus gets irritating.

I am unable to check whether the game has downloaded or not, as the orgress suddenly stops at around 17-19% e.g.

I download a game: Persona 4 Golden >> Size is 3GB+ >> ETA: say about 29 mins >> PROGRESS STARTS >> PROGRESS @ 17% Loader Freezes completely, but download is continuing in the background >> only issue is you can not determine when the download is going to be completed as the progress bar/completion status has all but frozen.

Please check and fix.

**Comment by RY0M43CH1Z3N** [comment]
I downloaded Resident Evil 2 and i don't have this error

**Comment by diddyrips5000** [comment]
It happens, I will upload a picture that I took from the Vita itself as proof, its been pissing me off as its been over a few months since v.5 was released, and been using this every time its been happening on games over a specified size.

How I do this:

WiFi ON >> Launch PKGI >> Select any game over 1.5GB >> start download >> leave console while it dwnloads >> PKGI progress bar suspends/hangs (ETA/Download Speed/%/green bar all stop) but actual download still goes on without me knowing that its still in progress.

Console is not on idle mode, I have turned the time saver etc all off, this does not affect the PKGI bug above regardless.

Try downloading the following games it always happens on these:

P4G
Ys VIII
Yakuza
Trails Series
and so on...

basically any download thats in the 2GB - 4GB size area

**Comment by millenius** [comment]
same issue here as well. I let P4G download overnight, came back and it was finished. I then started D4:TNP (1.11GB) and the screen froze at 15% while the write access LED on my flash drive is still flashing very rapidly. What should I do? ETA says 35 minutes should I just close pkgi after an hour or so... or is my only option to press circle and cancel it?

**Comment by ghamson** [comment]
I find that when this happens to me... it is absolutely still downloading and will eventually install once this complete... however pressing circle and starting the download again carries on where it left off

**Comment by millenius** [comment]
> however pressing circle and starting the download again carries on where it left off
Thanks for that tip 👍 
My question also answered itself; once the download finished, it went straight from the 15% bar to saying
Successfully installed, press cross to close.

**Comment by MarioMasta64** [comment]
alot of times it will continue but the bar wont change, its mostly a cosmetic issue. its still downloading.
